playbooks/scenario4_setup.yml

```diff
@@ -77,21 +77,62 @@
     dnsname_or_ip: "{{ hostvars.bastion.ansible_host }}"
     group: "scn4_ecs"
 
-- name: Configure cron on main host for stress testing
+- name: Checking load processes
+  hosts: localhost
+  become: yes
+  tasks:
+    - name: Get reporting processes list from remote host  # noqa 301 305
+      ignore_errors: yes
+      shell: "ps aux | grep 'cpu_load' | awk '{print $2}'"
+      register: process_monitor
+
+    - name: Kill running reporting processes  # noqa 301 305
+      ignore_errors: yes
+      shell: "kill {{ process_monitor.stdout_lines[0] }}"
```

Comment:
`|quote` filter to avoid injections
https://docs.ansible.com/ansible/latest/modules/shell_module.html